### DISCUSSION TITLE
Is there an alternative to the Native Assets? A managed version of libSkiaSharp.dll

### DISCUSSION THREAD

**Discussion by naufalAcunomic** [mono/SkiaSharp#2297] [Q&A]
Hi all, I am currently developing an extension to an already existing app called Acumatica. However, it has compatibility issues with libSkiaSharp.dll because it is an unmanaged .dll file. Acumatica only "likes" managed .dll(s) but I really need to use SkiaSharp as the library I am currently using is dependent on it. Does anyone know how to get SkiaSharp.dll to work without using unmanaged libSkiaSharp.dll, is there a managed version of the .dll? 

Please forgive me if none of this makes sense, I am new to programming. If there is anything I can do to clear things up a bit more I am more than willing. Any information/help is deeply appreciated.

**Comment by neodynamic** [comment]
SkiaSharp is a .NET managed wrapper of native Skia libs. If you need pure .NET managed graphic lib then use .NET System.Drawing classes or WPF instead

**Reply by naufalAcunomic** [reply]
I see, thank you kindly for the help.
